xml/docu_styleguide.language.xml

```diff
@@ -468,16 +469,17 @@
   <para>
    Use quotations to quote from sources, such as books.
    In other cases, avoid using quotation marks.
-   Do not use irony and do not use quotation marks to mark irony.
-   In the case of computer input, computer output, and user
-   interface elements, use more appropriate markup. See also
+   For computer input, computer output, and user
+   interface elements, use different markup. See also
    <xref linkend="sec-command-line"/> and
    <xref linkend="sec-ui-item-markup"/>.
+   Also, do nor use quotations marks to mark irony. Avoid irony in
+   technical writing.
   </para>
   <para>
    To create quotations, use the
    <tag class="emptytag">quote</tag>
-   element, as it is easier to localize than hardcoded quotation marks and
+   element. It is easier to localize and
```

Comment:
Using a comparative ("easier") but removing the object you're comparing to ("hardcoded quotation marks") makes no sense to me.

Reply:
Fixed, but split the sentence in to two smaller units anyway.